### DISCUSSION TITLE
Group work

### DISCUSSION THREAD

**Discussion by 777ParallelClematis** [Isaac-Smit/Conway-s-Game-of-Life-Group-1-#5] [General]
Hey all, I wont be able to be at the planned group session today at 12, because I have my CSA due at 5. I'll be in the fishbowl tomorrow if we'd like to work together there.

**Comment by 777ParallelClematis** [comment]
Hey all, what is our plan? @Isaac-Smit

**Comment by 777ParallelClematis** [comment]
Hey all @Isaac-Smit @WilliamBus @Gourav-sharmaa 
Tomorrow after class I am going to see what I can get done of the assessment, even if its just notes in English. Join me if you wish, I'll be in the fishbowl.

**Comment by WilliamBus** [comment]
Notification?

**Comment by WilliamBus** [comment]
@777ParallelClematis notifcations?

**Reply by 777ParallelClematis** [reply]
Notified

**Comment by 777ParallelClematis** [comment]
@Isaac-Smit could you send an invite request to Noah? Just to mark it off. Ta
